### ISSUE TITLE
Django 1.8 gives  RemovedInDjango19Warning

### ISSUE THREAD

**Issue by abdulhaq-e** [django-extensions/django-extensions#597]
When I use shell_plus, I receive warnings of code that will be deprecated in django 1.9.

It may not be as important as other issues right now but I just thought of listing them here as a reminder. The first and last warnings may not be related to django-extensions.

```
..../django/django/core/management/base.py:259: RemovedInDjango19Warning: "requires_model_validation" is deprecated in favor of "requires_system_checks".
  RemovedInDjango19Warning)

..../lib/python2.7/site-packages/django_extensions/management/shells.py:97: RemovedInDjango19Warning: The utilities in django.db.models.loading are deprecated in favor of the new application loading system.
  from django.db.models.loading import get_models, get_apps

..../lib/python2.7/site-packages/django_extensions/management/shells.py:136: RemovedInDjango19Warning: [a.models_module for a in get_app_configs()] supersedes get_apps().
  for app_mod in get_apps():

..../django/django/utils/lru_cache.py:101: RemovedInDjango19Warning: The app_mod argument of get_models is deprecated.
  result = user_function(*args, **kwds)
```

**Comment by trbs** [comment]
Focus right now it to make sure we support Django 1.8 when it comes out.
After that we should fix these before 1.9 comes out :)

**Comment by blueyed** [comment]
Could be closed as duplicate of #661.
